### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rwols**
Mention that the user requires a Node.js installation.

### CODE REVIEW

**Review by jfcherng — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ Python support for Sublime's LSP plugin provided through [microsoft/pyright](htt
 
 ## Installation
 
-1. Install [LSP](https://packagecontrol.io/packages/LSP) and
+1. Make sure you have Node.js installed and `node` is in your `$PATH`. The language server subprocess is a Node.js app.
```
Comment (jfcherng) [inline README.md:7]: Will it be good to link Node.js to https://nodejs.org/?

**Comment by rchl** [comment]
lsp_utils posts something in the console but I guess it's not obvious. That's why I'd like to be able to trigger a output panel with a message for better discoverability.

And I guess I can already do that but ideally there should be an official API in LSP for that.

**Comment by rwols** [comment]
> lsp_utils posts something in the console but I guess it's not obvious.

We need both. It should be mentioned in the readme as well as a runtime check.

>  there should be an official API in LSP for that.

I had something for that https://github.com/sublimelsp/LSP/commit/a78e9582a1bd638a90475722ba4bf069a1195b24 but I didn't finish it. Maybe you can pick it up?

File: README.md (hunk 1 above)
Comment (Ultra-Instinct-05) [inline README.md:7]: As per `plugin.py`, the minimum required version of node seems to be `12.0.0`. Maybe worth adding it to avoid ambiguity.

**Pull request merged**